docs/reference/utilities/text.md

```diff
@@ -0,0 +1,247 @@
+# Typography
+
+Usual text utility classes to overcome the common text management needs.
+
+## Text Alignment
+
+<br> Is used for aligning the inner content of a block element.
+
+| Class         | Property             |
+| ------------- | ---------------------|
+| u-text-left   | `text-align: left`   |
+| u-text-center | `text-align: center` |
+| u-text-right  | `text-align: right`  |
+| u-text-justify| `text-align: justify`|
+
+### Left
+<br> Use `u-text-left` to aligns the text to the left:
+
+```html
+<p class="u-text-branding-primary u-text-left">Hello my name is Normandy!</p>
+```
+
+<pre>
+    <p class="u-text-branding-primary u-text-left">Lorem ipsum dolor sit amet, consectetuer adipiscing elit.<br>Aenean commodo ligula eget dolor.<br>Aenean massa. Cum sociis natoque penatibus et magnis dis<br>parturient montes,nascetur ridiculus mus. Donec quam felis,<br>ultricies nec, pellentesque eu, pretium quis, sem.
+    </p>
+</pre>
+
+---
+
+### Center
+<br> Use `u-text-center` to center the text:
+
+```html
+<p class="u-text-branding-primary u-text-center">Hello my name is Normandy!</p>
```

Comment:
Podem fer servir una frase una mica més... professional? XD

O un lorem ipsum per defecte, o la mítica que es fa servir per les tipografies (perquè té tots els caracters): "The quick brown fox jumped over the lazy dog."